cdc/http_handler.go

```diff
@@ -68,6 +79,18 @@ func handleOwnerResp(w http.ResponseWriter, err error) {
 	writeData(w, commonResp{Status: true})
 }
 
+func handleOwnerRespJSON(w http.ResponseWriter, err error) {
+	if err != nil {
+		if errors.Cause(err) == concurrency.ErrElectionNotLeader {
+			writeErrorJSON(w, http.StatusBadRequest, err)
```

Comment:
Is this case tested?

Reply:
Only do manual testing.
Not unit test yet, I will add.